internal/version/tool_latest_version_finder.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2022 MongoDB Inc
```

Comment:
> the whole code in tool_latest_version_finder.go is not mocked, and it is tested in builder_test.go

Are this test direct or indirect? for direct tests a test file `tool_latest_version_finder_test.go` should have those tests, if those are indirect tests then my main point we lack direct testing of `tool_latest_version_finder_test` but again since most of the logic is searching/filtering, examples type of tests are better than unit testing

Reply:
moved to another package andded tests, thx!